Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -3510,18 +3510,40 @@ fileprivate final class TokenStreamCreator: SyntaxVisitor {
   }
 }
 
-private func isNestedInPostfixIfConfig(node: Syntax) -> Bool {
-    var this: Syntax? = node
+private func isNestedInPostfixIfConfig(node: MemberAccessExprSyntax) -> Bool {
+  func containsDescendent(ancestor: Syntax, node: MemberAccessExprSyntax) -> Bool {
+    if ancestor.children(viewMode: .sourceAccurate).contains(Syntax(node)) {
+      return true
+    }
 
-    while this?.parent != nil {
-      if this?.parent?.is(PostfixIfConfigExprSyntax.self) == true {
+    for child in ancestor.children(viewMode: .sourceAccurate) {
+      if containsDescendent(ancestor: child, node: node) {
         return true
       }
-
-      this = this?.parent
     }
 
     return false
+  }
+
+  var this: Syntax? = Syntax(node)
+
+  while this?.parent != nil {
+    if this?.is(TupleExprElementSyntax.self) == true {
```

Comment:
This handles the scenario [in this test](https://github.com/apple/swift-format/pull/402/files#diff-e366fdb32497e39bd5e8e8f666541fb18fbded18ad436459b26665555fd1e190R427)

Reply:
This is the most fragile part which I dislike the most. It handles the scenario in that test fine, but what other scenarios are there that this does not handle, or that this breaks?